templates/hotgraphicPopup.jsx

```diff
@@ -37,6 +37,32 @@ export default function HotgraphicPopup(props) {
         aria-hidden={!_isActive ? true : null}
         >
 
+          {(_imageAlignment === 'left' || _imageAlignment === 'top') &&
+            <div className={classes([
```

Comment:
Minor: Is it worth putting this image container div in it's own partial as it is repeated below?

Reply:
We have a partial for this already in the core:

https://github.com/adaptlearning/adapt-contrib-core/blob/master/templates/image.jsx

See adapt-contrib-graphic for use:

https://github.com/adaptlearning/adapt-contrib-graphic/blob/446c14f4674274c01902999593959bde1d2bcef2/templates/graphic.jsx#L46-L55

Reply:
Seeing as this PR is based on the [`_imageAlignment`](https://github.com/adaptlearning/adapt-contrib-core/blob/535905f91b1dfca190179f6e6ef2909e020f0efb/templates/notifyPopup.hbs#L34) implementation in Core Notify, if we want to use a partial here shouldn't we do the same for `notifyPopup.hbs` for consistency?

Reply:
The https://github.com/adaptlearning/adapt-contrib-core/blob/master/templates/image.jsx partial is only for the img tag in jsx. 

The _imageAlignment behaviour is where to put the img tag in the dom. 

The notifyPopup template is in handlebars, which doesn't have an equivalent img partial.

Reply:
Using this pr https://github.com/adaptlearning/adapt-contrib-core/pull/479 you should be able to specify:
```js
<templates.image {..._graphic} 
classNamePrefixSeparator='__item-'
classNamePrefixes={['component-item', 'hotgraphic-item']} 
attributionClassNamePrefixes={['component', 'hotgraphic']} 
/>
```